### ISSUE TITLE
AgentID.get(NamedParameter) does not work

### ISSUE THREAD

**Issue by ettersi** [org-arl/fjage#284]
This works:
```groovy
> req = new ParameterReq(node).get(new NamedParameter("nodeName"))
ParameterReq[nodeName:?]

> node.request(req)
ParameterRsp[nodeName:A]
```

This doesn't:
```groovy
> node.get(new NamedParameter("nodeName"))
```

Based on my understanding of the code, the two should be exactly the same:
https://github.com/org-arl/fjage/blob/454ca543e83c82cf7e8cfc0632c28b7a1cd55a92/src/main/java/org/arl/fjage/AgentID.java#L189-L214

**Comment by ettersi** [comment]
Found it! The problem is the `rsp.get()`:

```groovy
> param = new NamedParameter("nodeName")
nodeName

> req = new ParameterReq(node).get(param)
ParameterReq[nodeName:?]

> rsp = node.request(req)
ParameterRsp[nodeName:A]

> rsp.get(param) == null
true
```

Specifically, the problem is this line:

https://github.com/org-arl/fjage/blob/454ca543e83c82cf7e8cfc0632c28b7a1cd55a92/src/main/java/org/arl/fjage/param/ParameterRsp.java#L95

```groovy
> rsp.param == param
false

> rsp.param.class
class org.arl.unet.nodeinfo.NodeInfoParam

> param.class
class org.arl.fjage.param.NamedParameter
```

**Comment by ettersi** [comment]
Logging this here so I don't forget. 

@mchitre requested that a fully qualified `NamedParameter` should match precisely the parameter with the same fully qualified name. This will require changes beyond the change to `ParameterRsp.get()` mentioned above. Specifically, this line here will need to be updated:

https://github.com/org-arl/fjage/blob/454ca543e83c82cf7e8cfc0632c28b7a1cd55a92/src/main/java/org/arl/fjage/param/ParameterMessageBehavior.java#L199

**Comment by mchitre** [comment]
Implemented in #285
